2_run_inference/2_deploy_model.ipynb

```diff
@@ -48,16 +49,17 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "c03b56d7",
    "metadata": {},
    "source": [
     "Now that you have trained successfully your model, you want to look at the predictions on some sample images. To do so, you will need to find the s3 path of the exported model. You can navigate to the Training jobs section of the AWS web UI and click on the training job of interest. Scroll down and you should see something like this:\n",
     "\n",
-    "![Example Artefact](../data/example_artefact.png)\n",
+    "![Example Artifact](../data/example_artefact.png)\n",
```

Comment:
@ruddyscent Could you please change the file name to match your edit? example_**artifact**.png

Reply:
I've incorporated your suggestion.